integration/setup/key.go

```diff
@@ -0,0 +1,19 @@
+// +build k8srequired
+
+package setup
```

Comment:
I think this should be a separate `key` package.

Reply:
Sorted. I will improve the `env` stuff in a future PR. It is a bit all over the place currently.